app/services/data_migrations/backfill_current_course_option_id.rb

```diff
@@ -0,0 +1,17 @@
+module DataMigrations
+  class BackfillCurrentCourseOptionId
+    TIMESTAMP = 20210419154319
+    MANUAL_RUN = false
+
+    def change
+      sql = <<~POPULATE_CURRENT_OPTION_ID_SQL.squish
+        UPDATE application_choices
+           SET current_course_option_id = COALESCE(offered_course_option_id, course_option_id)
```

Comment:
I don't _think_ we need `WHERE current_course_option_id IS NULL` as this will always give us the correct response (ie we've always populating both fields since https://github.com/DFE-Digital/apply-for-teacher-training/pull/4546/files)?

Reply:
I extracted this from https://github.com/DFE-Digital/apply-for-teacher-training/pull/4499 which stops writing to `offered_course_option_id`, where this `WHERE` made more sense.

Reply:
Having said that, I can't actually see this `WHERE` clause in the migration!

Reply:
Haha sorry, I was just imagining it for argument's sake!